standalone-metastore/metastore-server/src/main/sql/hive/upgrade.order.hive

```diff
@@ -1,6 +1,6 @@
 3.0.0-to-3.1.0
-3.1.0-to-4.0.0-alpha-1.hive
-4.0.0-alpha-1-to-4.0.0-alpha-2.hive
-4.0.0-alpha-2-to-4.0.0-beta-1.hive
-4.0.0-beta-1-to-4.0.0.hive
-4.0.0-to-4.1.0.hive
\ No newline at end of file
+3.1.0-to-4.0.0-alpha-1
```

Comment:
hive suffix is optional, right, same thing is in master now

Reply:
The suffix was unnecessarily extra. It was the main reason why the upgrade script failed.
https://issues.apache.org/jira/browse/HIVE-28352

```
Underlying cause: java.io.FileNotFoundException : /opt/hive/scripts/metastore/upgrade/hive/upgrade-3.1.0-to-4.0.0-alpha-1.hive.hive.sql (No such file or directory)
```